starter-code/css/main.css

```diff
@@ -6,3 +6,137 @@ Green: #00B172
 White: #FFF
 
 */
+* {
+  border: solid transparent 1px;
+}
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  margin: 0 auto;
+  width: 1400px;
```

Comment:
When it comes to your body, don't use absolute value like px, because as soon as you will open the website on an other computer it may not respond as expected, instead try with a 100% :)